README.md

```diff
@@ -159,6 +159,207 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
 
 The resulting model will be saved to the `hf-ner-model/` directory.
 
+## `textcat.openai.correct`: Textcat annotation with zero- or few-shot learning
```

Comment:
As of [April last year](https://github.com/explosion/prodigy-openai-recipes/pull/54) Github can generate a table of contents for a README. 

Given the amount of recipes, we can start thinking about a hierarchy here. Maybe on a high level have NER/TEXTCAT? 

![CleanShot 2023-01-31 at 10 17 04](https://user-images.githubusercontent.com/1019791/215719263-02c44b3a-3fb8-4717-b0de-4a6a2ba85d54.png)

We can also choose to list our own table of contents on the readme itself. This might make sense as this document is pretty long now.

Reply:
I like that idea too. There are not nearly as many recipes as in prodigy-recipes repo, but it would help the navigation for sure.

Reply:
That definitely sounds good!